Source/Python/Particles/MultiParticleContainer.cpp

```diff
@@ -22,25 +22,39 @@ void init_MultiParticleContainer (py::module& m)
         )
 
         .def("set_plasma_lens_strength",
```

Comment:
There should be two versions of set_plasma_lens_strength, the original one with two arguments and the new one with four.